src/dlls/mscoree/mscoree.cpp

```diff
@@ -181,21 +175,12 @@ BOOL WINAPI DllMain(HANDLE hInstance, DWORD dwReason, LPVOID lpReserved)
             {
                 return FALSE;
             }
-
-#ifdef FEATURE_MERGE_JIT_AND_ENGINE
```

Comment:
The closed build still define FEATURE_MERGE_JIT_AND_ENGINE and statically links the JIT. I think this will break JIT initialization for that case.  You may need to keep this but update it to call jitStartup.